File: array_max.c
// 数组中最大元素
void main() {
    int arr[] = {7, 14, 2, 8, 1};
    int n = 5;
    int max = arr[0];

    for (int i = 1; i < n; i++) {
        if (arr[i] > max) {
            max = arr[i];
        }
    }
}

File: array_max.rs
fn main() {
    let arr = [7, 14, 2, 8, 1];
    let n = 5;
    let mut max = arr[0];

    for i in 1..n {
        if arr[i] > max {
            max = arr[i];
        }
    }
}
